internal/identity/oidc/azure/azure.go

```diff
@@ -8,15 +10,22 @@ import (
 	"net/url"
 	"time"
 
-	oidc "github.com/coreos/go-oidc"
+	go_oidc "github.com/coreos/go-oidc"
 	"golang.org/x/oauth2"
 
 	"github.com/pomerium/pomerium/internal/httputil"
+	"github.com/pomerium/pomerium/internal/identity"
+	"github.com/pomerium/pomerium/internal/identity/oidc"
 	"github.com/pomerium/pomerium/internal/log"
 	"github.com/pomerium/pomerium/internal/sessions"
 	"github.com/pomerium/pomerium/internal/version"
 )
 
+var _ identity.Authenticator = &AzureProvider{}
+
+// Name identifies the Azure identity provider
```

Comment:
@desimone, thinking we can do what I suggested here too to make our code consistent.
https://github.com/pomerium/pomerium/pull/560/files#r398807570